helm/thingsboard/values.yaml

```diff
@@ -119,6 +125,8 @@ jsexecutor:
     #server: docker.io
     #pullPolicy: Always
     #tag: "X.Y.Z"
+  logger:
+    level: warn
```

Comment:
capitalized?

Reply:
it seems that jsexecutor uses lower case for logger, I suppose is something about javascript or nodejs, I'm  not sure.

Reply:
interesting...